lib/commands/find.js

```diff
@@ -116,6 +131,48 @@ helpers.findNativeElementOrElements = async function (strategy, selector, mult,
   }
 };
 
+helpers.getFirstVisChild = async function (mult, context) {
+  log.info(`Getting first visible child`);
+  if (mult) {
+    throw new Error("Cannot get multiple first visible children!");
+  }
+  if (!context) {
+    throw new Error("Cannot get first visible child without a context element");
+  }
+  let index = 1;
+  // loop through children via class-chain finds, until we run out of children
+  // or we find a visible one
+  while (true) { // eslint-disable-line no-constant-condition
```

Comment:
This might be an endless loop if there are no visible children

Reply:
I don't think so. At some point the class chain query `*[${index}]` will throw an error because an element will not be found at that index. it will result in an element not found error to the client, which is what we want (no visible child element was found)

Reply:
can it be added to the comment? Otherwise it's not very clear, that we expect an exception there.